examples/test/DatePicker_test.js

```diff
@@ -32,3 +32,41 @@ test("should be able to set from and to dates", async t => {
         .expect(xAxisLabels.textContent)
         .eql("FebMarAprMayJunJulAugSepOct");
 });
+
+test("should be able to set from and to months", async t => {
```

Comment:
It seems that tests in this file follow a different convention that other TestCafe tests :( See the previous comment about the uppercase letter in the TestCafe test.

Reply:
There is a typo _amd_ -> _and_.